### ISSUE TITLE
Global TinyMCE configuration is not respected

### ISSUE THREAD

**Issue by tomgross** [plone/plone.app.mosaic#336]
In Plone 5.x it is possible to add custom JSON configuration to TinyMCE in the control panel. The TinyMCE used in Mosaic does not respect this setting but should do it IMO.

Steps to reproduce:
1. Add a JSON configuration to the TinyMCE control panel. Something like

 ```
{
    "paste_as_text": true
}
``` 

will do.
2. Paste some fancy HTML text to a document to a document page. The HTML formating will be stripped out.
3. Paste some fance HTML text to the Mosaic HTML tile. The HTML formating IS NOT striped out but SHOULD.

**Comment by datakurre** [comment]
Thanks. It should work. I use it every day with import css plugin. I need to try this out myself. For example, it could be possible that we still read the result HTML from TinyMCE using a wrong API so that the filtering does not apply...

**Comment by datakurre** [comment]
@tomgross Confirmed that this setting was explicitly overwritten. My pull should fix this in 2.0.0, because that stil remains unreleased :)

**Comment by datakurre** [comment]
Note that you can also add explicit remove fomat action

```
  <records interface="plone.app.mosaic.interfaces.IFormat"
           prefix="plone.app.mosaic.richtext_toolbar.removeformat">
    <value key="name">toolbar-removeformat</value>
    <value key="category">text</value>
    <value key="label">Remove Format</value>
    <value key="action">removeformat</value>
    <value key="icon">false</value>
    <value key="favorite">false</value>
    <value key="weight">330</value>
  </records>

  <record name="plone.app.mosaic.app_tiles.plone_app_standardtiles_html.available_actions">
    <value purge="false">
      <element>toolbar-removeformat</element>
    </value>
  </record>

  <record name="plone.app.mosaic.widget_actions.plone_app_z3cform_widget_RichTextFieldWidget.actions">
    <value purge="false">
      <element>toolbar-removeformat</element>
    </value>
  </record>
```
